ui/app/mainui/AppMain.qml

```diff
@@ -651,6 +651,15 @@ Item {
                     }
                 }
 
+                
+                ModuleWarning {
+                    id: downloadingArchivesBanner
+                    Layout.fillWidth: true
+                    active: communitiesPortalLayoutContainer.communitiesStore.downloadingCommunityHistoryArchives
+                    type: ModuleWarning.Danger
+                    text: qsTr("Downloading message history archives, DO NOT CLOSE THE APP until this banner disappears.")
```

Comment:
Just a small UI/wording mismatch; the user actually _may_ close the app (window in fact, it will minimize to tray); however what must _not_ be done is `quit`

Reply:
I'll defer to @John-44 on this one

Reply:
Discussed offline, @John-44 is fine with leaving the text as is now